static/styles/zulip.css

```diff
@@ -2831,12 +2853,34 @@ ul.expanded_subjects {
     padding-bottom: 2px;
 }
 
+ul.expanded_private_messages {
+    list-style-type: none;
+    font-weight: 300;
+    font-size: 84%;
+    margin-left: 0px;
+    padding-bottom: 2px;
+}
+
 li.show-more-topics,
 li.expanded_subject {
     position: relative;
     padding-left: 33px;
 }
 
+
+li.show-more-private-messages,
+li.expanded_private_message {
+    position: relative;
+    padding-left: 24px;
+    padding-bottom: 1px;
+    padding-top: 2px;
+}
+
+
+#global_filters li:hover {
+    background-color: #e2e8dd;
+}
```

Comment:
What's the story with this last block of the CSS change?  Seems like it's maybe somewhat orthogonal to the feature?